src/pods/configure-verbs/configure-verbs.component.tsx

```diff
@@ -53,6 +73,22 @@ export const ConfigureVerbsComponent: React.FC<Props> = props => {
       </Button>
 
       <ul>
+        <li>
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={temporalSelected !== 0}
+                onChange={handleMasterCheckboxChange()}
+                color="primary"
+                indeterminate={
+                  temporalSelected !== 0 &&
+                  temporalSelected !== verbCollection.length
+                }
```

Comment:
Same thing this we can enclose this logic check inside the hooks and provide some meaningful name, e.g.

indetermainate{SelectionMatchesItemsChecked()}